### PR TITLE
Revert to previous file structure to avoid doubling up on pages.

### PR DESCRIPTION
**Pull request by djhavert**
Commit https://github.com/jetstream-cloud/js2docs/commit/f19455b901a37086cf2e309aa83e1d5ef81b49fb move /ui/cli to /cli to fix the accordioning of the nav bar. It's currently unclear whether flattening the file structure will help with the accordion, so in the meantime let's revert back to the original so we do not have a double copy of every file between /ui/cli and /cli.

Note: After the commit above that changed the file structure, the only other change to these sets of files came from https://github.com/jetstream-cloud/js2docs/commit/d444a7ac70010a5b7af8c43ba6de6033a41bfe26. This change has been copied to the original file at /ui/cli/auth.md, so as far as I am aware things should be back to normal and all caught up with changes made after the files structure change.

### CODE REVIEW

File: docs/general/terraform.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ git clone git@github.com:wellsaar/terraform-jetstream.git
 This command creates a clone of the git repo that contains the code examples that we use below. The master branch of the git repo above should be a good starting point in developing your own terraform code.
 
 # Running Terraform
-Make sure that you have setup and downloaded your openrc file. [Setting up the openrc.sh for the Jetstream2 CLI](../../ui/cli/auth/#setting-up-application-credentials-and-openrcsh-for-the-jetstream2-cli)
+Make sure that you have setup and downloaded your openrc file. [Setting up the openrc.sh for the Jetstream2 CLI](../ui/cli/auth.md#setting-up-application-credentials-and-openrcsh-for-the-jetstream2-cli)
```
Comment (djhavert) [inline docs/general/terraform.md:38]: Leave as original. This change belongs to a different issue.
```suggestion
Make sure that you have setup and downloaded your openrc file. [Setting up the openrc.sh for the Jetstream2 CLI](../../ui/cli/auth/#setting-up-application-credentials-and-openrcsh-for-the-jetstream2-cli)
```

Comment (djhavert) [inline docs/general/terraform.md:38]:
```suggestion
Make sure that you have setup and downloaded your openrc file. [Setting up the openrc.sh for the Jetstream2 CLI](../ui/cli/auth/#setting-up-application-credentials-and-openrcsh-for-the-jetstream2-cli)
```

File: docs/general/terraform.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ git clone git@github.com:wellsaar/terraform-jetstream.git
 This command creates a clone of the git repo that contains the code examples that we use below. The master branch of the git repo above should be a good starting point in developing your own terraform code.
 
 # Running Terraform
-Make sure that you have setup and downloaded your openrc file. [Setting up the openrc.sh for the Jetstream2 CLI](../../ui/cli/auth/#setting-up-application-credentials-and-openrcsh-for-the-jetstream2-cli)
+Make sure that you have setup and downloaded your openrc file. [Setting up the openrc.sh for the Jetstream2 CLI](../ui/cli/auth/#setting-up-application-credentials-and-openrcsh-for-the-jetstream2-cli)
```
Comment (djhavert) [inline docs/general/terraform.md:38]:
```suggestion
Make sure that you have setup and downloaded your openrc file. [Setting up the openrc.sh for the Jetstream2 CLI](../../ui/cli/auth/#setting-up-application-credentials-and-openrcsh-for-the-jetstream2-cli)
```

**Comment by jlf599** [comment]
Looks like this should undo the previous flattening. Going to merge.

**Pull request merged**